### PR TITLE
route fixes

### PR DESCRIPTION
**Pull request by kurund**
## Summary

* Clean up the route implementation

## Test Plan

* Not needed

### CODE REVIEW

**Comment by kurund** [comment]
@rathorevaibhav sorry forgot to comment earlier, tested few scenarios pending scenarios, and is working as expected. Ready for your review.

File: src/App.tsx [hunk 1]
```diff
@@ -29,20 +29,21 @@ const App = () => {
...
     }
 
-  // For logout acion, we don't need to check if the user is logged in or not. Hence, adding it at top level
-  routes = (
-    <Switch>
-      <Route path="/logout" exact component={Logout} />
-      {routes}
-    </Switch>
-  );
+    // For logout acion, we don't need to check if the user is logged in or not. Hence, adding it at top level
```
Comment (rathorevaibhav) [inline src/App.tsx:39]:
```suggestion
    // For logout action, we don't need to check if the user is logged in or not. Hence, adding it at top level
```

File: src/App.tsx [hunk 2]
```diff
@@ -29,20 +29,21 @@ const App = () => {
 
   let routes;
 
-  if (authenticated !== undefined)
+  if (authenticated !== undefined) {
     if (authenticated) {
       routes = <AuthenticatedRoute />;
     } else {
       routes = <UnauthenticatedRoute />;
     }
```
Comment (rathorevaibhav) [inline src/App.tsx:37]: Very minor thing. Do you think something like this is valid?
```suggestion
  if (authenticated !== undefined && authenticated) {
    routes = <AuthenticatedRoute />;
  } else {
      routes = <UnauthenticatedRoute />;
  }
```

**Review by rathorevaibhav — COMMENTED** [review]
one question and one minor feedback. @kurund have a look

File: src/App.tsx (hunk 2 above)
Comment (kurund) [reply src/App.tsx:37]: @rathorevaibhav Actually we don't want to assign any routes, hence the current logic. In the above scenario, we will set UnAuthenticated routes.

Comment (rathorevaibhav) [reply src/App.tsx:37]: Sounds good then.

**Review by rathorevaibhav — APPROVED** [review]

**Pull request merged**